src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3VettingHelper.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.helpers;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.collections4.CollectionUtils;
+import uk.gov.hmcts.ecm.common.helpers.UtilHelper;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.et.common.model.ccd.items.RespondentSumTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.types.RespondentSumType;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.NO;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.YES;
+
+@Slf4j
+public class Et3VettingHelper {
+
+    private Et3VettingHelper() {
+        //Access through static methods
+    }
+
+    public static String getEt3Dates(CaseData caseData) {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        stringBuilder.append("| Dates| |\r\n");
+        stringBuilder.append("|--|--|\r\n");
+        stringBuilder.append(String.format("|ET1 Served| %s|\r\n", formatDate(caseData.getClaimServedDate())));
```

Comment:
If "Served" is capitalised then shouldn't "due" and "received" be also (or vice-versa)?